array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,19 @@
+/* exported drop */
+// count the indexes
+// for loop which initialization starts at the count value
+// push the new values into a new array and return it
+
+function drop(array, count) {
+  if (array === 0) {
+    return array;
+  }
```

Comment:
an array won't ever equal a number, but that's ok, you can remove this `if` statement